### DISCUSSION TITLE
哥特动漫王国 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#142] [模板讨论]
哥特动漫王国 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by wsdjj** [comment]
日志：Failed at 1/5 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 503: Service Unavailable,
Request URL: https://www.gtloli.gay/forum.php

**Comment by wsdjj** [comment]
大佬，我运行显示503错误，设置了代理也一样

**Reply by wjf0214** [reply]
503是服务器错误，不过网站正常访问，还是看看网络设置之类的有没有问题？

**Comment by biyafanqiang** [comment]
大佬们，现在哥特是啥情况？上一年12月突然进不去了，打开完全白色无反应，就连导航页都一样
